worker/games.py

```diff
@@ -106,10 +106,16 @@ def setup(item, testing_dir):
   else:
     raise Exception('Item %s not found' % (item))
 
-def setup_engine(destination, worker_dir, sha, repo_url, concurrency):
-  if os.path.exists(destination): os.remove(destination)
+def setup_engine(engine_dir, sha, repo_url, concurrency):
+  new_engine_name = 'stockfish_' + sha
+  destination = os.path.join(engine_dir, new_engine_name)
+  if os.path.exists(engine_dir):
```

Comment:
Leftover of the original code: if `engine_dir` exist we don't call `setup_engine()`